hivemind_core/database.py

```
@@ -263,6 +263,10 @@ def __iter__(self) -> Iterable['Client']:
         """
         pass
 
+    def sync(self):
+        """update db from disk if needed"""
+        pass
```

Ruff (0.8.2)

`AbstractDB.sync` is an empty method in an abstract base class, but has no abstract decorator

(B027)